src/rpc/net.cpp

```diff
@@ -531,7 +533,10 @@ static UniValue getnetworkinfo(const JSONRPCRequest& request)
     obj.pushKV("timeoffset",    GetTimeOffset());
     if (node.connman) {
         obj.pushKV("networkactive", node.connman->GetNetworkActive());
-        obj.pushKV("connections",   (int)node.connman->GetNodeCount(CConnman::CONNECTIONS_ALL));
+        obj.pushKV("connections", (int)node.connman->GetNodeCount(CConnman::CONNECTIONS_ALL));
+        obj.pushKV("connections_in", (int)node.connman->GetNodeCount(CConnman::CONNECTIONS_IN));
+        obj.pushKV("connections_out", (int)node.connman->GetNodeCount(CConnman::CONNECTIONS_OUT));
+
```

Comment:
nit: Any reason to add an empty line?

Reply:
Added a line before and after for readability as it's a subgroup of 5 lines building a separate UniValue object rather than just one line.

Reply:
That's strange and inadvertant. Will remove.

Reply:
Thanks -- removed.